test/sparse_variational.jl

```diff
@@ -131,7 +131,7 @@
                 isapprox(
                     elbo(SparseVariationalApproximation(Centered(), fz, q_ex), fx, y),
                     logpdf(fx, y);
-                    atol=1e-6,
+                    atol=1e-5,
```

Comment:
any idea why you had to loosen the tolerance?

Reply:
I'd be pro reverting it :+1: broken nightly is a common issue anyways; I'd prefer more trust (due to tighter tolerances) in the stable builds

Reply:
```suggestion
                    atol=1e-6,
```